### ISSUE TITLE
w3.org-link instead of content

### ISSUE THREAD

**Issue by BinkaDroid** [friendica/friendica#7013]
### Expected behavior
content/ message

### Actual behavior
A link like `https://www.w3.org/ns/activitystreams#Announce`; correct content appears some time later but additionally to the strange announce-post

### Steps to reproduce the problem
no steps, started happening a few days ago, other people on other friendica servers having the same problem
### Friendica version you encountered the problem
2019.03

Example Screenshot (left source, right what is displayed in my friendica)
![Screenshot 2019-04-14 at 22 21 37](https://user-images.githubusercontent.com/39274494/56098757-30845e80-5f04-11e9-98bf-9f61e502571b.png)

**Comment by tobiasd** [comment]
> no steps, started happening a few days ago, other people on aother
> friendica servers having the same problem

This is a glitch in the Matrix ;-) The current development version of Friendica has a new way to display when a posting was reshared. This is essentially what the message displayed with the link is, the announcement that somebody reshared the posting.

So, not a bug, but the dawning of a new feature / era in Friendica 2019.06.

**Comment by BinkaDroid** [comment]
To understand it correctly: this happens, when an original message is delivered from a server running any dev-version of friendica?

I'm asking, because in this example, two comments via diaspora still aren't showing up even 3 & 4 hours later:
![Screenshot_20190415-093925](https://user-images.githubusercontent.com/39274494/56115231-3a897a00-5f63-11e9-816a-432deb460aab.png)

**Comment by annando** [comment]
You see this "2 people reshared this" on the left side. This is what the "Announce" message is for.

**Comment by BinkaDroid** [comment]
Awhhhhh, sorry, I'm a little bit slow sometimes... Now I understand it (haven't had another look at the original posting).

**Comment by loma-one** [comment]
> no steps, started happening a few days ago, other people on aother friendica servers having the same problem
> This is a glitch in the Matrix ;-) The current development version of Friendica has a new way to display when a posting was reshared. This is essentially what the message displayed with the link is, the announcement that somebody reshared the posting. So, not a bug, but the dawning of a new feature / era in Friendica 2019.06.

But this is what is displayed on the Stable Friendica 2019.03. This is rather irritating.

**Comment by tobiasd** [comment]
> But this is what is displayed on the Stable Friendica 2019.03. This
> is rather irritating.

Yes, because the stable Friendica version does not know a better way of how to deal with this comment. Therefore it displays the "content" of the comment. The development version of Friendica is able to understand the comment, and displays the meaning of it accordingly.

So stable does see echos from the future. Hence my analogy with the glitch in the matrix.

**Comment by BinkaDroid** [comment]
As a workaround, I changed the CSS for the default theme in my friendica installation:
```
a[href="https://www.w3.org/ns/activitystreams#Announce"]:before {
	color: #999;
	font-style: italic;
	content: "shared this/ hat diesen Beitrag geteilt" !important;
}
a[href="https://www.w3.org/ns/activitystreams#Announce"] {
	color:transparent!important;
}
```
so it looks like this:
![Anmerkung 2019-04-15 215223](https://user-images.githubusercontent.com/39274494/56160988-c681bd00-5fc8-11e9-821d-1cad3300982b.jpg)

If you cannot fiddle around in your friendica-installation, you can use the extension "stylus" (NOT: stylish) - available for Vivaldi, Chrome, Firefox etc. & and use this custom CSS to make this glitch less annoying ;-)

**Comment by annando** [comment]
I think we can close this issue, since it had been a side effect that had been caused by a new development and not a bug?

**Comment by MrPetovan** [comment]
Yes, we won't able to fix it back in 2019.03 anyway.
